src/openrct2/Editor.cpp

```diff
@@ -128,7 +128,7 @@ namespace OpenRCT2::Editor
         ToolCancel();
         auto intent = Intent(WindowClass::Loadsave);
         intent.PutExtra(INTENT_EXTRA_LOADSAVE_TYPE, LOADSAVETYPE_LOAD | LOADSAVETYPE_GAME);
-        intent.PutExtra(INTENT_EXTRA_CALLBACK, reinterpret_cast<void*>(ConvertSaveToScenarioCallback));
+        intent.PutExtra(INTENT_EXTRA_CALLBACK, reinterpret_cast<close_callback>(ConvertSaveToScenarioCallback));
```

Comment:
Likewise, would you mind renaming this `CloseCallback` to follow current code style conventions?